### ISSUE TITLE
Confirm-URI fails with powershell v5.1

### ISSUE THREAD

**Issue by LexiconCode** [jtmoon79/PythonEmbed4Win#1]
- To reproduce the bug.
1. `.\PythonEmbed4Win.ps1`
2. See error

- Error 
URI `https://www.python.org/ftp/python` downloaded to `C:\Users\MainLaptop\AppData\Local\Temp\python-latest-amd64\python_versions.html`
```
C:\Users\MainLaptop\Desktop\New2\PythonEmbed4Win.ps1 : at Confirm-URI, 
C:\Users\MainLaptop\Desktop\New2\PythonEmbed4Win.ps1: line 378
at Confirm-URI-Python-Version, C:\Users\MainLaptop\Desktop\New2\PythonEmbed4Win.ps1: line 405
at Scrape-Python-Versions, C:\Users\MainLaptop\Desktop\New2\PythonEmbed4Win.ps1: line 494    
at <ScriptBlock>, C:\Users\MainLaptop\Desktop\New2\PythonEmbed4Win.ps1: line 676
at <ScriptBlock>, <No file>: line 1
At line:1 char:1
+ . 'C:\Users\MainLaptop\Desktop\New2\PythonEmbed4Win.ps1'
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,PythonEmbed4Win.ps1  
 
PS C:\Users\MainLaptop>
C:\Users\MainLaptop\Desktop\New2\PythonEmbed4Win.ps1 : A parameter cannot be found that matches      
parameter name 'SkipHTTPErrorCheck'.
At line:1 char:1
+ . 'C:\Users\MainLaptop\Desktop\New2\PythonEmbed4Win.ps1'
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,PythonEmbed4Win.ps1  
```
- debugging info
![image](https://github.com/jtmoon79/PythonEmbed4Win/assets/24551569/74051ed5-066e-4b6e-b826-bb8dd19b0aaf)

- Environment 
```
Edition	Windows 10 Home
Version	22H2
OS build	19045.2965
PSVersion   5.1.19041.2673
```

**Comment by LexiconCode** [comment]
Looks like powershell v5.1 does not have parameter name 'SkipHTTPErrorCheck' for `Invoke-WebRequest`
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-webrequest?view=powershell-5.1

Works with powershell v7.2 and above

**Comment by LexiconCode** [comment]
It may be worth supporting 5.x with https://stackoverflow.com/questions/19122378/powershell-web-request-without-throwing-exception-on-4xx-5xx

Edit: After implementing the above work around there are other aspects of the code that are incompatible with v5.1.

**Comment by jtmoon79** [comment]
Fixed in #2 
Thanks @LexiconCode !
